developer-extension/src/background/domain/syncRules.ts

```diff
@@ -62,11 +62,11 @@ function buildRules(
     const devRumUrl = useRumSlim ? DEV_RUM_SLIM_URL : DEV_RUM_URL
     logger.log('add redirect to dev bundles rules')
     rules.push(
-      createRedirectRule(/^https:\/\/.*\/datadog-rum(-v\d|-canary|-staging)?\.js$/, { url: devRumUrl }),
-      createRedirectRule(/^https:\/\/.*\/datadog-rum-slim(-v\d|-canary|-staging)?\.js$/, {
+      createRedirectRule(/^https:\/\/.*\/datadog-rum(-v\d|-canary|-staging|-v\d-canary)?\.js$/, { url: devRumUrl }),
```

Comment:
**💬 suggestion:** ‏I think we should simplify the regex to make it future proof: `datatdog-rum(-\w+)?\.js` . It's pretty low-risk.

Reply:
💭 Though I think there is an upside of having the explicit rules: it can serve as docs for the versions we have and always updated.

Reply:
We have better documentation here though (although a bit outdated): https://datadoghq.atlassian.net/wiki/spaces/RUMP/pages/2523531139/SDK+distribution#CDN

Reply:
fixed, Mainly for simplification as I'm not totally convinced that it's futureproof, especially with the split bundles initiative, but we will fix it in due time.

I'll update the doc too, thanks